docs/tutorial/tut_as_of_date.md

```diff
@@ -0,0 +1,42 @@
+An As of Dates table contains a single column of dates used to construct the history in tables like PITs and Bridges. 
+
+The date range and the date interval size is often informed by the current reporting period and can be changed on the fly.
+An example of a usual date range could be the _end of day_ timestamps for each day of the last 3 months.
+
+Periodically, the As of Dates table would be refreshed, to accommodate for the new reporting period.
+
+### Structure
+
+The As of Dates table consists of a single datetime column, and is generated by the 
+[dbt_utils.date_spine](https://github.com/dbt-labs/dbt-utils#date_spine-source) macro.
+
+!!! note
+    dbt_utils is another package for dbt, provided by dbt Labs. The date_spine macro above is provided by this package.
+    For more information, go read the date_spine docs [here](https://github.com/dbt-labs/dbt-utils#date_spine-source).
+    dbtvault includes dbt_utils as a dependency, and is installed alongside the dbtvault package. 
+
+The macro will require the following input:
+
+#### Date interval (datepart)
+
+#### Start Date (start_date)
+
+#### End Date (end_date)
+
+### Creating the model
+
+Create a new model as before. We'll call this one `as_of_dates`.
+
+=== "as_of_dates.sql"
+
+    ```jinja
+    {{ dbt_utils.date_spine(
```

Comment:
Have you checked that this works?

Also, I removed the above about the column name defaulting to "AS_OF_DATE" because the model above does not actually do that, it just generates a "DATE" column.

Reply:
You're right. I was not clear in my description.

The date_spine macro does not name the generated column AS_OF_DATE but just DATE, but the column needs to be (re)named AS_OF_DATE as 

The work around to rename the column to AS_OF_DATE is to wrap the date_spine macro call in a CTE or to create our own `as_of_date` macro that calls `date_spine`.

Reply:
Resolved